src/main/java/com/github/manolo8/darkbot/gui/drawables/TrailDrawer.java

```diff
@@ -44,8 +44,9 @@ private void drawTrail(MapGraphics mg) {
 
         if (distance > 500) {
             last.setTo(hero);
-        } else if (distance > 10) {
-            positions.add().init(last, last.setTo(hero));
+        } else if (distance > 25) {
+            positions.add().init(last, hero);
+            last.setTo(hero);
```

Comment:
```suggestion
            positions.add().init(last, last.setTo(hero));
```
any reason not to leave it as it is?